OvmfPkg/Library/PlatformBootManagerLib/BdsPlatform.c

```diff
@@ -539,6 +539,8 @@ PlatformBootManagerBeforeConsole (
     ConnectVirtioPciRng,
     NULL
     );
+
+  gDS->Dispatch ();

```

Comment:
Should we include this fix also in `ConnectVirtioPciRng()` in `OvmfPkg/Library/PlatformBootManagerLibBhyve/BdsPlatform.c` ?

Reply:
Probably a good idea. I'll open a separate PR for that later.